apps/web/tsconfig.e2e.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
I was thinking of moving e2e tests to its own package to avoid this. WDYT?

Reply:
nice idea to clean up the mess with cjs/mjs